actionpack/lib/action_controller/metal/etag_with_template_digest.rb

```diff
@@ -25,7 +25,7 @@ module EtagWithTemplateDigest
       class_attribute :etag_with_template_digest
       self.etag_with_template_digest = true
 
-      ActiveSupport.on_load :action_view, yield: true do
+      ActiveSupport.on_load :action_controller, yield: true do
```

Comment:
Could you apply those two patches in our application?
[patches.zip](https://github.com/rails/rails/files/874347/patches.zip)

We will need one commit from https://github.com/rails/rails/pull/28402 too, but I'll make sure we merge that PR before this PR.